DATA/production/calib/tpc-laser-aggregator.sh

```diff
@@ -20,7 +20,7 @@ publish_after=440
 min_tracks=0
 num_lanes=36
 
-
+REMAP="--condition-remap \"file:///home/wiechula/processData/inputFilesTracking/triggeredLaser/=GLO/Config/GRPECS;GLO/Config/GRPMagField;TPC/Calib/LaserTracks\" "
```

Comment:
Not sure, but according to @shahor02  comment this should be
```suggestion
REMAP="--condition-remap \"file:///home/wiechula/processData/inputFilesTracking/triggeredLaser/=GLO/Config/GRPECS,GLO/Config/GRPMagField,TPC/Calib/LaserTracks\" "
```
So `,` instead of `;`. @shahor02 , can you confirm?

Reply:
Yes, object paths should be separated by comma.

Reply:
Thanks. I changed it.

Reply:
Thanks for the confirmation